server/server.go

```diff
@@ -90,6 +107,16 @@ func New(cfg Config) (*Server, error) {
 		countCallNotFound:      &metrics.LabelMap{Label: "method"},
 		countCallInternalError: &metrics.LabelMap{Label: "method"},
 	}
+
+	if cfg.BackupBucket != "" {
```

Comment:
Maybe do this below so that the setup/start are in a single check?

Reply:
Done (moved the goroutine start into the existing block, I figured we should only mutate the input mux once we're happy that no further errors are happening)